insecure_generate_tls.sh

```diff
@@ -11,6 +11,7 @@ KEYFILE="${KANI_TMP}key.pem"
 CERTFILE="${KANI_TMP}cert.pem"
```

Comment:
Should there be a / between the env var and the cert?

Reply:
`KANI_TMP=/tmp/kanidm/` above handles it